### DISCUSSION TITLE
Publish to Vercel failure

### DISCUSSION THREAD

**Discussion by tarikrital** [swellstores/community#523] [Help]
Welcome,

Using Horizon theme gives the errors below on production and local `vercel dev`.

![image](https://github.com/swellstores/community/assets/52907282/2160bdf4-db7f-464b-ae58-b9e370ff7eaa)

![image](https://github.com/swellstores/community/assets/52907282/adf79dd0-8467-44a9-b53e-2e59d508c32c)

**Comment by logeshswell** [comment]
Can you share the below information please?
1. request logs in **Network** tab.
2. Server logs from localhost.
